imagehash/tests/test_hex_conversions.py

```diff
@@ -0,0 +1,99 @@
+import unittest
+import numpy as np
+import imagehash
+
+
+# Each row is a test case where the first value is a bit sequence and
+# the second value is the expected hexadecimal representation for it.
+binary_to_hexadecimal_values = [
+	['1', '1'],
+	['11', '3'],
+	['111', '7'],
+	['1111', 'f'],
```

Comment:
@JohannesBuchner , this test case covers hash_size=2. Previously there was no output. This test case assert there is a output and it is correct.